model/src/main/java/com/javahelp/model/user/User.java

```diff
@@ -0,0 +1,84 @@
+package com.javahelp.model.user;
+import com.javahelp.model.user.UserInfo;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.UserType;
+import com.javahelp.model.user.ProviderUserInfo;
+
+/**
+ * A class for a user entity
+ */
+public class User {
+
+    /**
+     * --- Attributes---
+     * stringID - the ID of the user account
+     * userInfo - all information of the user. It varies based on Client/Provider
+     */
+    private String stringID;
+    private UserInfo userInfo;
+    private String username;
```

Comment:
Same thing as before with Javadocs formatting. For private fields, Javadocs are not necessary. In this case, you could either turn these into comments on the fields themselves, or remove the comment entirely.